resolver/resolver.cc

```diff
@@ -453,6 +453,7 @@ class ResolveConstantsWalk {
             }
         } else {
             ENFORCE(resolved.data(ctx)->isClassOrModule());
+            job.klass.data(ctx)->unsetClassLinearizationComputed();
```

Comment:
could there be a helper method that does `unsetClassLinearizationComputed && >mixins().emplace_back(resolved)`?

This will ensure that we never get it wrong.